OpenStack-Rabbit-Consumer/rabbit_consumer/message_consumer.py

```diff
@@ -110,7 +122,7 @@ def check_machine_valid(rabbit_message: RabbitMessage) -> bool:
         )
         return False
 
-    if not is_aq_managed_image(rabbit_message):
+    if not is_aq_managed_image(vm_data):
         logger.debug("Ignoring non AQ Image: %s", rabbit_message)
```

Comment:
Is `rabbit_message` still meant to be here if it's been replaced by `vm_data` in the line above?